settings.gradle.kts

```diff
@@ -16,3 +16,4 @@ include("network-client-default")
 include("network-client-okhttp")
 include("pubsub-adapter")
 include("live-objects")
+include("example")
```

Comment:
Is this supposed to be just an `example` module? Are we planning to include all types of examples in this section?

Reply:
In such a case, we can call it `examples` since it can include both `java` and `android` examples.

Reply:
renamed